### ISSUE TITLE
Error: Inference status for the task XXX is failed. requests.exceptions.HTTPError: 408 Client Error: Request Timeout for url: http://nuclio:8070/api/function_invocations

### ISSUE THREAD

**Issue by ZzuMrW** [opencv/cvat#4950]
### My actions before raising this issue
- [x] Read/searched [Semi-automatic and Automatic Annotation](https://opencv.github.io/cvat/docs/administration/advanced/installation_automatic_annotation/)

### Expected Behaviour
When I run automatic annotation, if the model is running by cpu, doesn't have any wrong, but running by gpus, it's has wrong like this:
![Timeout](https://user-images.githubusercontent.com/12119047/190120360-beba2f13-f616-4467-8eba-948c122cc0e4.png)

#### nuctl get function
```
NAMESPACE |                       NAME                        | PROJECT | STATE | NODE PORT | REPLICAS  
  nuclio    | openvino-dextr                                    | cvat    | ready |     49153 | 1/1       
  nuclio    | openvino-omz-public-faster_rcnn_inception_v2_coco | cvat    | ready |     49211 | 1/1       
  nuclio    | openvino-omz-public-yolo-v3-tf                    | cvat    | ready |     49154 | 1/1       
  nuclio    | tf-faster-rcnn-inception-v2-coco-gpu              | cvat    | ready |     57253 | 1/1       
  nuclio    | tf-matterport-mask-rcnn-cpu                       | cvat    | ready |     32001 | 1/1     
```

#### the logs of cvat server:
[cvat_server.log](https://github.com/opencv/cvat/files/9565048/cvat_server.log)

#### the logs of nuclio-nuclio-tf-faster-rcnn-inception-v2-coco-gpu
[faster_rcnn_inception_v2_coco_gpu.log](https://github.com/opencv/cvat/files/9565054/faster_rcnn_inception_v2_coco_gpu.log)

### Your Environment

- cvat_version: v2.1.0、develop
- Docker version `docker version` (e.g. Docker 17.0.05):  20.10.17
- Are you using Docker Swarm or Kubernetes? docker
- Operating System and version (e.g. Linux, Windows, MacOS): 
    ```
      lsb_release -a
      Distributor ID:	Ubuntu
      Description:	Ubuntu 18.04.6 LTS
      Release:	18.04
      Codename:	bionic
    ```

**Comment by bsekachev** [comment]
Hello, try to provide us some logs from nuclio-dashboard container). I expect they have some error details.

**Comment by ZzuMrW** [comment]
> Hello, try to provide us some logs from `nuclio-dashboard` container). I expect they have some error details.
> 
> ```shell
>  docker logs nuclio-dashboard 
> ```

Hi, I rebuild cvat and reproduce the error. cvat server and gpu model's error no change.  this is the nuclio dashboard's log. 

[nuclio.log](https://github.com/opencv/cvat/files/9573951/nuclio.log)

**Comment by bsekachev** [comment]
@ZzuMrW 

Have you done preparation steps? https://opencv.github.io/cvat/docs/administration/advanced/installation_automatic_annotation/#gpu-support

Can you submit that GPUs are available inside a docker container?

**Comment by ZzuMrW** [comment]
> @ZzuMrW
> 
> Have you done preparation steps? https://opencv.github.io/cvat/docs/administration/advanced/installation_automatic_annotation/#gpu-support
> 
> Can you submit that GPUs are available inside a docker container?

@bsekachev 
```shell
docker run --gpus all --rm cvat.tf.faster_rcnn_inception_v2_coco_gpu:latest nvidia-smi
```
![image](https://user-images.githubusercontent.com/12119047/190539247-4070c4f1-06d8-46d3-891e-ede5dc23cb2c.png)

```shell
docker run --gpus all -it --rm cvat.tf.faster_rcnn_inception_v2_coco_gpu:latest python -c "import tensorflow as tf; print(tf.reduce_sum(tf.random.normal([1000, 1000])))"
```
```
2022-09-16 02:03:29.834759: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libnvinfer.so.6
2022-09-16 02:03:29.836226: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libnvinfer_plugin.so.6
2022-09-16 02:03:30.277668: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2022-09-16 02:03:30.289847: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 0 with properties: 
pciBusID: 0000:19:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.290383: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 1 with properties: 
pciBusID: 0000:1a:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.291096: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 2 with properties: 
pciBusID: 0000:67:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.291642: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 3 with properties: 
pciBusID: 0000:68:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.291716: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2022-09-16 02:03:30.291784: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10
2022-09-16 02:03:30.294149: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10
2022-09-16 02:03:30.294708: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10
2022-09-16 02:03:30.297185: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10
2022-09-16 02:03:30.298683: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10
2022-09-16 02:03:30.298747: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2022-09-16 02:03:30.301308: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1697] Adding visible gpu devices: 0, 1, 2, 3
2022-09-16 02:03:30.301794: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2022-09-16 02:03:30.313703: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3699850000 Hz
2022-09-16 02:03:30.314584: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x41cb190 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2022-09-16 02:03:30.314610: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2022-09-16 02:03:30.753713: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x3c57610 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2022-09-16 02:03:30.753762: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): NVIDIA GeForce RTX 3080, Compute Capability 8.6
2022-09-16 02:03:30.753785: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (1): NVIDIA GeForce RTX 3080, Compute Capability 8.6
2022-09-16 02:03:30.753831: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (2): NVIDIA GeForce RTX 3080, Compute Capability 8.6
2022-09-16 02:03:30.753853: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (3): NVIDIA GeForce RTX 3080, Compute Capability 8.6
2022-09-16 02:03:30.754947: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 0 with properties: 
pciBusID: 0000:19:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.755470: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 1 with properties: 
pciBusID: 0000:1a:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.755887: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 2 with properties: 
pciBusID: 0000:67:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.756283: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 3 with properties: 
pciBusID: 0000:68:00.0 name: NVIDIA GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.71GHz coreCount: 68 deviceMemorySize: 9.78GiB deviceMemoryBandwidth: 707.88GiB/s
2022-09-16 02:03:30.756339: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2022-09-16 02:03:30.756364: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10
2022-09-16 02:03:30.756402: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10
2022-09-16 02:03:30.756430: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10
2022-09-16 02:03:30.756459: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10
2022-09-16 02:03:30.756483: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10
2022-09-16 02:03:30.756509: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2022-09-16 02:03:30.760853: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1697] Adding visible gpu devices: 0, 1, 2, 3
2022-09-16 02:03:30.761057: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2022-09-16 02:06:58.517978: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1096] Device interconnect StreamExecutor with strength 1 edge matrix:
2022-09-16 02:06:58.518003: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1102]      0 1 2 3 
2022-09-16 02:06:58.518012: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] 0:   N N N N 
2022-09-16 02:06:58.518019: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] 1:   N N N N 
2022-09-16 02:06:58.518026: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] 2:   N N N N 
2022-09-16 02:06:58.518032: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] 3:   N N N N 
2022-09-16 02:06:58.519148: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1241] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8169 MB memory) -> physical GPU (device: 0, name: NVIDIA GeForce RTX 3080, pci bus id: 0000:19:00.0, compute capability: 8.6)
2022-09-16 02:06:58.519800: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1241] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 9037 MB memory) -> physical GPU (device: 1, name: NVIDIA GeForce RTX 3080, pci bus id: 0000:1a:00.0, compute capability: 8.6)
2022-09-16 02:06:58.520346: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1241] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:2 with 9037 MB memory) -> physical GPU (device: 2, name: NVIDIA GeForce RTX 3080, pci bus id: 0000:67:00.0, compute capability: 8.6)
2022-09-16 02:06:58.520883: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1241] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:3 with 8525 MB memory) -> physical GPU (device: 3, name: NVIDIA GeForce RTX 3080, pci bus id: 0000:68:00.0, compute capability: 8.6)
tf.Tensor(-378.90915, shape=(), dtype=float32)
```

**Comment by ZzuMrW** [comment]
> @ZzuMrW
> 
> Have you done preparation steps? https://opencv.github.io/cvat/docs/administration/advanced/installation_automatic_annotation/#gpu-support
> 
> Can you submit that GPUs are available inside a docker container?

@bsekachev Hello, I tried change this model's (faster_rcnn_inception_v2_coco) base images from `tensorflow/tensorflow:2.1.1-gpu` to `tensorflow/tensorflow:latest-gpu`, it's works and has no error, so I guess it's maybe related to the base images.
And then I also tried to run another tensorflow model (matterport) , but it not works, the error similar to this issue. I guess it's maybe related to tensorflow version and the base imags.
